### ISSUE TITLE
[patch.crates-io]

### ISSUE THREAD

**Issue by cemoktra** [pkgw/cranko#31]
Hi,

i just played around with your project to see if it might help deploying a monorepo but it seems the you tools ignores the `[patch.crates-io]` block

**Comment by pkgw** [comment]
That's right. No good reason for that — I just didn't think of that case. I'm quite tied up these days so it would probably be a while before I get a chance to look at this myself, but pull requests would definitely be welcome!
